eng/common/scripts/job-matrix/Create-PrJobMatrix.ps1

```diff
@@ -64,9 +64,7 @@ $packageProperties = Get-ChildItem -Recurse "$PackagePropertiesFolder" *.json `
 
 # set default matrix config for each package if there isn't an override
 $packageProperties | ForEach-Object {
-  if (-not $_.CIMatrixConfigs) {
```

Comment:
I blindly took this and it _definitely_ doesn't work. It's because `CIMatrixConfigs` isn't null, it's Empty Array.